automatic/1password8/1password8.nuspec

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>1password8</id>
+    <version>0.0</version>
+    <title>1Password</title>
+    <authors>AgileBits</authors>
+    <owners>chocolatey-community, AgileBits</owners>
```

Comment:
Just out of curiosity, as I've not seen anything from them since I last enquired, have AgileBits expressed further interest?

Reply:
I have not heard anything. The only one having contact with them was you, so if you haven't heard anything then I would say no.